akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -283,6 +283,17 @@ private[akka] object Running {
           // ignore
           this // it will be stopped by the first WriteMessageFailure message; not applying side effects
 
+        case DeleteMessagesSuccess(toSequenceNr) ⇒
```

Comment:
Not convinced these two Delete responses belong here but it's the only `JournalProtocol.Response` type function. I guess we could add a specific new function to handle them, aside from the Write commands.

Reply:
Maybe rename if to `onJournalResponse` then?

One thought, is there any guarantee the event sourced actor is in `Running` state when the ack/nack on a delete comes back?

Reply:
it can also be in `PersistingEvents` since delete operations will occur in the background without stashing

Reply:
Yes, well so far my problem is the `DeleteMessagesSuccess` returns to a self that ends up going to deadletters vs in Running where I can continue handling the processing.